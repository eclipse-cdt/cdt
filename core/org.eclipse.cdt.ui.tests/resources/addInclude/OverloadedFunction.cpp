#include "A.h"

void test() {
  func(0);
}
