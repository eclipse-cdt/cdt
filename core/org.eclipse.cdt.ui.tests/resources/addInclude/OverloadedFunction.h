namespace ns3 {

void func(int p);
void func(char* p);

}