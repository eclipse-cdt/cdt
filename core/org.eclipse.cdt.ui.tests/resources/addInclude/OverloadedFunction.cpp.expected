#include "OverloadedFunction.h"

#include "A.h"

using ns3::func;

void test() {
  func(0);
}
